src/smart-components/portfolio/portfolio-items.js

```diff
@@ -68,7 +71,9 @@ const PortfolioItems = ({
     'catalog:order_processes:link'
   ]);
   const dataSet = data ? data : results;
-  const metaInfo = meta ? meta : { count };
+  const metaInfo = meta ? meta : { count, limit, offset };
+  console.log('Debug - metaInfo: ', metaInfo);
```

Comment:
Do we want to leave this here for now or was it unintentional?

Reply:
I'll take it out